src/accel/cuda/GPUSolver.h

```diff
@@ -53,6 +53,7 @@
 /** Indexing macro for the angular fluxes for each polar angle and energy
  *  group for a given Track */
 #define boundary_flux(t,pe2) (boundary_flux[2*(t)*(*polar_times_groups)+(pe2)])
+#define start_flux(t,pe2) (start_flux[2*(t)*(*polar_times_groups)+(pe2)])
```

Comment:
Can you add a doxygen comment above `start_flux` akin to those for the other macros?